erpnext/erpnext_integrations/taxjar_integration.py

```diff
@@ -364,4 +376,28 @@ def get_line_items(doc):
 		}
 		line_items.append(line_item)
 
-	return line_items
\ No newline at end of file
+	return line_items
+
+def make_integration_request(doctype, docname, service=None, status="Queued", endpoint=None, error=None, output=None):
```

Comment:
This should be in frappe utils. 

@vishdha @sahil28297 @DeveloperMetal  whats your views?